.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,47 @@
+name: docker_release
+
+on:
+  push:
+    branches: [ dev ]
```

Comment:
We could also add `main` here if that's useful?

Reply:
I agree with the idea of adding main (in addition to dev) to trigger the publication of an image.
IMHO, 
- any push (merge) to dev should publish a docker image tagged as `dev` (which represents the latest stable enough version for testing, with the same semantic than a debian `testing` distribution.)
- any semver tag on the main branch should gnerate a ` stable` release, and this image should be published and tagged with the version number (semver). In addition to tagging with the version number, We may also add the tag `latest` so that this version is the one retrieved by default when a user wants to run the default Scaphandre, without providing explicitly the version in the docker command.
To get this differentiated behavior may involve 2 separate actions (one triggered on any push to dev branchh, and the other triggered by add a _git_ tag).

Reply:
@demeringo Thanks I added the main branch too.

For the `latest` tag this is set by the docker-metadata action when a new tag is pushed. If it's the latest according to semver. 

https://github.com/docker/metadata-action#latest-tag